controllers/binding/snapshotenvironmentbinding_controller.go

```diff
@@ -1,9 +1,10 @@
 /*
-Copyright 2023.
+Copyright 2022.
 Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+you may not use th
```

Comment:
Guessing this is an accidental deletion here.

Reply:
yes, yes it is!!